app/_src/gateway/breaking-changes/38x.md

```diff
@@ -15,4 +15,28 @@ Review the [changelog](/gateway/changelog/#3800) for all the changes in this rel
 
 ## Breaking changes and deprecations
 
-TBA
\ No newline at end of file
+### New DNS client
+Starting 3.8.x, a new DNS client library was implemented and added to {{site.base_gateway}}. The new DNS client introduces and new standardized way to configure a service, and helps improve performance. It is disabled by default, but will be enabled by default in a future release. For more information, see [Migrate to the new DNS client](/gateway/migrate-to-new-dns-client/).
+
+### `kong.logrotate` configuration file no longer overwritten during upgrade
+The `kong.logrotate` configuration file will no longer be overwritten during upgrade.
+When upgrading, set the environment variable `DEBIAN_FRONTEND=noninteractive` on Debian/Ubuntu to avoid any interactive prompts and enable fully automatic upgrades.
+
+### `log_statistics` defaults to `false` for AI Proxy
+A configuration validation was added to the [AI Proxy plugin](/hub/kong-inc/ai-proxy/) to prevent users from enabling `log_statistics` for
+providers that don't support statistics. In addition, the default of `log_statistics` was changed from
+`true` to `false`, and a database migration is added as well for disabling `log_statistics` if it
+has already been enabled upon unsupported providers.
+
+
+### Deprecations
+
+#### `sentinel_addresses` and `cluster_addresses` for Redis
+
+The following plugins switched `cluster_addresses` to `cluster_nodes` and `sentinel_addresses` to `sentinel_nodes` for Redis configuration:
+
+* [AI Rate Limiting Advanced](/hub/kong-inc/ai-rate-limiting-advanced/)
+* [GraphQL Proxy Caching Advanced](/hub/kong-inc/graphql-proxy-cache-advanced/)
+* [GraphQL Rate Limiting Advanced](/hub/kong-inc/graphql-rate-limiting-advanced/)
+* [Proxy Caching Advanced](/hub/kong-inc/proxy-cache-advanced/)
+* [Rate Limiting Advanced](/hub/kong-inc/rate-limiting-advanced/)
```

Comment:
Hi @jschmid1 ! I'm writing a doc that compiles the 3.8 breaking changes and I had a question about the `sentinel_addresses` and `cluster_addresses` for Redis feature changes you made. Are these converted automatically with 3.8 or does the user have to manually migrate or change anything? Thanks!